exec/java-exec/src/main/java/org/apache/drill/exec/planner/sql/parser/SqlRefreshMetadata.java

```diff
@@ -63,6 +76,8 @@ public SqlOperator getOperator() {
   public List<SqlNode> getOperandList() {
     List<SqlNode> ops = Lists.newArrayList();
     ops.add(tblName);
+    ops.add(allColumns);
+    ops.add(fieldList);
```

Comment:
You also need to amend `public void unparse(SqlWriter writer, int leftPrec, int rightPrec) {` based on existence of `fieldList`.

Reply:
Fixed